cpu/mips32r2_common/Makefile.include

```diff
@@ -1,18 +1,8 @@
 INCLUDES += -I$(RIOTCPU)/mips32r2_common/include
 
-export USEMODULE += mips32r2_common
-export USEMODULE += mips32r2_common_periph
-export USEMODULE += newlib
-
-# mips32 needs periph_timer for its gettimeofday() implementation
-export USEMODULE += periph_timer
-
 ifeq ($(USE_UHI_SYSCALLS),1)
   #Use UHI to handle syscalls
   LINKFLAGS += -luhi
-  export USEMODULE += newlib_syscalls_mips_uhi
+  USEMODULE += newlib_syscalls_mips_uhi
```

Comment:
This line should be removed, this was introduced by a wrong rebase

Reply:
you mean the exports had already been removed?

Reply:
There was an PR for removing the export, but this PR is moving the line in Makefile.dep. In the current state, the line is duplicated.

Reply:
Fixed now.